experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1358,6 +1404,169 @@ def test_serializer_feature_config_validation_treatment_value_no_schema(self):
+        reference_feature_value.value = """\
+            {
+            "bar": {
+                "baz": "baz",
+                "qux": 123
+            }
+            }
+        """.strip()
+        reference_feature_value.save()
+
+        serializer = NimbusReviewSerializer(
+            experiment,
+            data=NimbusReviewSerializer(
+                experiment,
+                context={"user": self.user},
+            ).data,
+            context={"user": self.user},
+        )
+        self.assertTrue(serializer.is_valid())
+
+    @mock.patch(
+        "experimenter.features.manifests.nimbus_fml_loader.NimbusFmlLoader.get_fml_errors",
+    )
+    def test_serializer_fml_invalid_schema(self, mock_get_fml_errors):
+        fml_errors = [
+            NimbusFmlErrorDataClass(
+                line=0,
+                col=0,
+                message="Incorrect value",
+                highlight="enabled",
+            ),
+            NimbusFmlErrorDataClass(
+                line=0,
+                col=0,
+                message="Type not allowed",
+                highlight="record",
+            ),
+        ]
+        mock_get_fml_errors.return_value = fml_errors
+
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.CREATED,
+            status=NimbusExperiment.Status.DRAFT,
+            application=NimbusExperiment.Application.FENIX,
+            channel=NimbusExperiment.Channel.RELEASE,
+            warn_feature_schema=False,
+            feature_configs=[
+                NimbusFeatureConfigFactory.create(
+                    application=NimbusExperiment.Application.FENIX,
+                    schemas=[
+                        NimbusVersionedSchemaFactory.build(
+                            version=None,
+                            schema=REF_JSON_SCHEMA,
+                        )
+                    ],
+                )
+            ],
+            is_sticky=True,
+            firefox_min_version=NimbusExperiment.MIN_REQUIRED_VERSION,
+        )
+        reference_feature_value = experiment.reference_branch.feature_values.get()
+        reference_feature_value.value = """\
+            {
+            "bar": {
+                "baz": "baz",
+                "qux": 123
+            }
+            }
+        """.strip()
+        reference_feature_value.save()
+
+        serializer = NimbusReviewSerializer(
+            experiment,
+            data=NimbusReviewSerializer(
+                experiment,
+                context={"user": self.user},
+            ).data,
+            context={"user": self.user},
+        )
+
+        self.assertFalse(serializer.is_valid())
+        mock_get_fml_errors.assert_called()
+        self.assertIn(
+            NimbusConstants.ERROR_FML_VALIDATION,
+            serializer.errors["reference_branch"]["feature_values"][0]["value"][0],
+        )
+        self.assertEqual(len(serializer.errors), 2)
```

Comment:
I would expect to actually validate that the FML error data we setup above is included in the response here.